lib/cookie.js

```diff
@@ -1006,6 +1050,14 @@ CookieJar.prototype.setCookie = function(cookie, url, options, cb) {
     loose = options.loose;
   }
 
+  var sameSiteContext = null;
```

Comment:
Should be `0` to agree with the other call site of `checkSameSiteContext` on line 1195

Reply:
nevermind... that one's `sameSiteLevel` not `...Context`